### PR TITLE
Amgeo/update luis authoring

### PR DESCRIPTION
**Pull request by AmirGeorge**
Update the LUIS CLI tool to pick up the latest swagger changes

### CODE REVIEW

File: packages/LUIS/bin/luis.js [hunk 1]
```diff
@@ -226,6 +226,9 @@ async function runProgram() {
                 case "sublist":
                     result = await client.model.addSubList(args.region, args.cloud, args.appId, args.versionId, args.clEntityId, requestBody, args);
                     break;
+                case "appazureaccount":
+                    result = await client.azureAccounts.assignToApp(args.region, args.cloud, args.appId, requestBody, args);
```
Comment (AmirGeorge) [inline packages/LUIS/bin/luis.js]: when testing, this gave an incorrectly formatted authorization header. Will fix it before merging.

File: packages/LUIS/bin/luis.js [hunk 2]
```diff
@@ -303,6 +323,19 @@ async function runProgram() {
                     throw new Error(`Unknown resource: ${target}`);
             }
             break;
+        // ------------------ PACKAGE ------------------
+        case "package":
+            switch (target) {
+                case "slot":
+                    result = await client.apps.packagePublishedApplicationAsGzip(args.region, args.cloud, args.appId, args.slotName, args);
```
Comment (AmirGeorge) [inline packages/LUIS/bin/luis.js]: the packaged file being outputted is corrupt, will fix it.

File: packages/LUIS/bin/luis.js (hunk 1 above)
Comment (AmirGeorge) [reply packages/LUIS/bin/luis.js]: resolved

File: packages/LUIS/bin/luis.js (hunk 2 above)
Comment (AmirGeorge) [reply packages/LUIS/bin/luis.js]: fixed

File: packages/LUIS/bin/luis.js [hunk 3]
```diff
@@ -227,7 +227,9 @@ async function runProgram() {
                     result = await client.model.addSubList(args.region, args.cloud, args.appId, args.versionId, args.clEntityId, requestBody, args);
                     break;
                 case "appazureaccount":
-                    result = await client.azureAccounts.assignToApp(args.region, args.cloud, args.appId, { azureAccountInfoObject: requestBody, ...args });
+                    var infoObj = {azureAccountInfoObject: requestBody};
+                    var options = Object.assign.apply(Object, [{}].concat(infoObj).concat(args));
```
Comment (omarsourour) [inline packages/LUIS/bin/luis.js]: `var options = Object.assign({}, { azureAccountInfoObject: requestBody }, args)`

File: packages/LUIS/bin/luis.js [hunk 4]
```diff
@@ -298,7 +300,9 @@ async function runProgram() {
                                 return;
                             }
                         }
-                        result = await client.azureAccounts.removeFromApp(args.region, args.cloud, args.appId, { azureAccountInfoObject: requestBody, ...args });
+                        var infoObj = {azureAccountInfoObject: requestBody};
+                        var options = Object.assign.apply(Object, [{}].concat(infoObj).concat(args));
```
Comment (omarsourour) [inline packages/LUIS/bin/luis.js]: Same here

File: packages/LUIS/bin/luis.js [hunk 5]
```diff
@@ -671,18 +674,44 @@ async function runProgram() {
...
+                var stream = result.readableStreamBody;
+                let body = [];
+                stream.on('readable', function() {
+                    let chunk;
+                    while ((chunk = stream.read()) != null) {
+                        body.push(chunk);
+                    }
+                });
+                stream.on('end', () => {
+                    body = body.slice(0, -1);
```
Comment (omarsourour) [inline packages/LUIS/bin/luis.js]: Why do you still need this?

**Pull request merged**